evap/staff/importers.py

```diff
@@ -244,7 +244,10 @@ def for_each_row_in_excel_file_do(self, row_function):
         for sheet in self.book.sheets():
             try:
                 for row in range(self.skip_first_n_rows, sheet.nrows):
-                    row_function(sheet.row_values(row), sheet, row)
+                    data = []  # container for normalized cell data
+                    for cell in sheet.row_values(row):
+                        data.append(' '.join(cell.split()))  # see https://stackoverflow.com/questions/2077897/substitute-multiple-whitespace-with-single-whitespace-in-python
```

Comment:
a list comprehension is probably more concise here.

Reply:
Yep, way better